programmers/자릿수-더하기.js

```diff
@@ -0,0 +1,8 @@
+function solution(n) {
+  const stringN = String(n);
+  let answer = 0;
```

Comment:
```suggestion
  const stringN = String(n);
  
  let answer = 0;
```
여기도 한 칸 띄워주세요.

풀이는 훌륭합니다 👏👏👍

Reply:
const와 let 사이가 아니라 let과 for문 사이를 띄우는 게 맞을까요...?

Reply:
아 맞아요 나 왜 저렇게 했지...?